### DISCUSSION TITLE
Reduce on the size of Move Column  to be equal to the move icon size   ( Drag/Drop column  )

### DISCUSSION THREAD

**Discussion by BenmoussaYounes** [KevinVandy/material-react-table#556] [Q&A]
I need to reduce the size of the Move Column so it take the icon size only

**Comment by KevinVandy** [comment]
https://www.material-react-table.com/docs/guides/display-columns#display-column-definition-options-prop

**Reply by BenmoussaYounes** [reply]
Thank you for replying, I really appreciate your help, 
I have already tried to use size of display columnDefOption props and set it to 0, 
    displayColumnDefOptions={{
            "mrt-row-drag": {
              header: false,
              size: 0, // ( 0 Is the max  i need somthing like -40 more of the 0 value ) 
            },
          }}
but it not enough if it's possible I need to reduce the size of that column so it takes exactly the move icon size (like - 35 more from 0 )
 hope u can help with this , thanks  for this awesome  table 🥇 

![move](https://github.com/KevinVandy/material-react-table/assets/108725492/f65dafd2-3bd0-4731-aac6-b2b40b4311a3)
